src/manager/auth.ts

```diff
@@ -133,14 +133,14 @@ export class ManagerAuth extends ApiConnection {
     username: string,
     password: string,
     seed: string[],
-  ): Promise<string> {
-    const data = await this.post<{jwt?: string}>('register', {
+  ): Promise<{jwt: string, backupId?: string}> {
+    const data = await this.post<{jwt?: string, backupId?: string}>('register', {
       seed,
       password,
       name: username,
     });
     if (!data.jwt) throw new Error('Failed to register new user.');
-    return data.jwt;
+    return data as {jwt: string, backupId?: string};
```

Comment:
This will break `runcitadel/dashboard/src/store/user.ts` `register` function which expects a string to be returned. Any change to this needs a corresponding change in `dashboard`.

Reply:
Yes, I will make a corresponding PR once this change is included in a release of this package.